src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -793,6 +793,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                         languageVersion = LanguageVersion.VisualBasic14
                                     Case "15", "15.0"
                                         languageVersion = LanguageVersion.VisualBasic15
+                                    Case "default"
```

Comment:
This is precisely the same as the C# command line option, and is intended to be a way for the user to select the default version, not the latest version. In this release they happen to be the same.